server/src/main/java/com/networknt/server/Server.java

```diff
@@ -538,4 +538,26 @@ private static void unzipFile(String path, String target) {
             logger.error("IOException", e);
         }
     }
+
+    // method used to merge status.yml and app-status.yml
+    protected static void mergeStatusConfig() {
+        Map<String, Object> appStatusConfig = Config.getInstance().getJsonMapConfigNoCache(STATUS_CONFIG_NAME[1]);
+        if (appStatusConfig == null) {
+            return;
+        }
+        Map<String, Object> statusConfig = Config.getInstance().getJsonMapConfig(STATUS_CONFIG_NAME[0]);
+        List<String> repeatStatusList = new ArrayList<>();
+        for (String key : appStatusConfig.keySet()) {
+            if (statusConfig.containsKey(key)) {
+                repeatStatusList.add(key);
+                logger.error("The status code: " + key + " has already in use by light-4j and cannot be overwritten," +
+                        " please change to another status code in app-status.yml if necessary.");
+            } else if (repeatStatusList.isEmpty()) {
```

Comment:
Also, I don't know if we care about the efficiency of this search too much.. but it might be cleaner if we used the retainAll method in Set to find the intersection, then just Map.addAll() if there isn't any.. doesn't seem like a necessary change but just in case

Reply:
@NicholasAzar Thank you for pointing out, they have been fixed.